rules/windows/persistence_remote_password_reset.toml

```diff
@@ -34,19 +36,30 @@ type = "eql"
 query = '''
 sequence by winlog.computer_name with maxspan=1m
   [authentication where event.action == "logged-in" and
+
     /* event 4624 need to be logged */
     winlog.logon.type : "Network" and event.outcome == "success" and source.ip != null and
     source.ip != "127.0.0.1" and source.ip != "::1" and
-    not winlog.event_data.TargetUserName : ("svc*", "PIM_*", "_*_", "*-*-*", "*$")] by winlog.event_data.TargetLogonId
+    not startsWith~(winlog.event_data.TargetUserName, "svc") and
+    not startsWith~(winlog.event_data.TargetUserName, "PIM_") and
+    not endsWith~(winlog.event_data.TargetUserName, "$") and
+    not (startsWith(winlog.event_data.TargetUserName, "_") and endsWith(winlog.event_data.TargetUserName, "_")) and
+    not winlog.event_data.TargetUserName regex "[a-zA-Z0-9]+-[a-zA-Z0-9]+-[a-zA-Z0-9]+"] by winlog.event_data.TargetLogonId
```

Comment:
What are we excluding here?

Reply:
the above are most noisy FPs based on service account naming convention